ci/linux-packages/apt/debian-bookworm/Dockerfile

```diff
@@ -58,6 +58,6 @@ RUN \
     python3-pip \
     tzdata \
     valac && \
-  pip3 install --upgrade meson && \
+  pip3 install --break-system-packages --upgrade meson && \
```

Comment:
We can use system Meson: https://github.com/apache/arrow/commit/41aedfdd4e452d547773dd2ea8a493139b24c572